package.json

```diff
@@ -9,7 +9,8 @@
   "dependencies": {
     "launchdarkly-react-native-client-sdk": "3.0.0",
     "react": "16.9.0",
-    "react-native": "0.61.2"
+    "react-native": "0.61.2",
+    "start": "^5.1.0"
```

Comment:
What is `start` used for?

Reply:
that was my bad, I do npm install start, and it installs start. I need to change the read me again   
 -_-